README.md

```diff
@@ -111,6 +111,18 @@ The spark-server module aims to provide a HTTP rest interface that is API compat
 programs you write to run against the Spark Cloud should also work on the Local Cloud.  Some features aren't here yet, but may be
 coming down the road, right now the endpoints exposed are:
 
+Claim Core
+
+`POST /v1/devices`
+
+Release Core
+
+`DELETE /v1/devices/:coreid`
+
+Provision Core and save Core's keys.
+
+`/v1/provisioning/:coreID`
+
 List devices
```

Comment:
don't like the fact that sometimes we use `Core` and another times `Device` for the same Entity.